logging-ktor-client/src/commonTest/kotlin/TuulboxLoggerTests.kt

```diff
@@ -0,0 +1,54 @@
+package com.juul.tuulbox.logging
+
+import com.juul.tuulbox.test.runTest
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.mock.MockEngine
+import io.ktor.client.engine.mock.respond
+import io.ktor.client.features.logging.LogLevel.ALL
+import io.ktor.client.features.logging.Logging
+import io.ktor.client.request.get
+import kotlin.test.AfterTest
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+private const val URL = "https://example.com"
```

Comment:
nit: Should we use something that _guarantees_ we don't talk over the wire?

```suggestion
private const val URL = "https://localhost"
```